pontoon/base/migrations/0090_auto_20170413_2101.py

```diff
@@ -0,0 +1,220 @@
+    u'sv': u'sv-SE',
+    u'th': u'th',
+    u'tlh': u'',  # MISSING?
+    u'tlh-Qaak': u'',  # MISSING?
+    u'tr': u'tr',
+    u'uk': u'uk',
+    u'ur': u'ur',
+    u'vi': u'vi',
+    u'yua': u'',  # MISSING?
+    u'zh-CHS': u'zh-CN',  # CONFLICT?
+    u'zh-CHT': u'zh-CN'  # CONFLICT
+}
+
+TERMINOLOGY_LOCALES = {
+    u'af-za': u'af',
+    u'am-et': u'am',
+    u'ar-sa': u'ar',
+    u'as-in': u'as',
+    u'be-by': u'be',
+    u'bg-bg': u'bg',
+    u'ca-es': u'ca',
+    u'cs-cz': u'cs',
+    u'cy-gb': u'cy',
+    u'da-dk': u'da',
+    u'de-de': u'de',
+    u'el-gr': u'el',
+    u'et-ee': u'et',
+    u'eu-es': u'eu',
+    u'fa-ir': u'fa',
+    u'fi-fi': u'fi',
+    u'fr-fr': u'fr',
+    u'ga-ie': u'ga-IE',
+    u'gl-es': u'gl',
+    u'gu-in': u'gu-IN',
+    u'ha-Latn-ng': u'ha',
+    u'he-il': u'he',
+    u'hi-in': u'hi-IN',
+    u'hr-hr': u'hr',
+    u'hu-hu': u'hu',
+    u'hy-am': u'hy-AM',
+    u'id-id': u'id',
+    u'ig-ng': u'ig',
+    u'is-is': u'is',
+    u'it-it': u'it',
+    u'ja-jp': u'ja',
+    u'ka-ge': u'ka',
+    u'kk-kz': u'kk',
+    u'km-kh': u'km',
+    u'kn-in': u'kn',
+    u'ko-kr': u'ko',
+    u'lo-la': u'lo',
+    u'lt-lt': u'lt',
+    u'lv-lv': u'lv',
+    u'ml-in': u'ml',
+    u'mr-in': u'mr',
+    u'ne-np': u'ne-NP',
+    u'nl-nl': u'nl',
+    u'pa-in': u'pa-IN',
+    u'pl-pl': u'pl',
+    u'ro-ro': u'ro',
+    u'ru-ru': u'ru',
+    u'si-lk': u'si',
+    u'sk-sk': u'sk',
+    u'sl-si': u'sl',
+    u'sq-al': u'sq',
+    u'sv-se': u'sv-SE',
+    u'te-in': u'te',
+    u'th-th': u'th',
+    u'tr-tr': u'tr',
+    u'uk-ua': u'uk',
+    u'ur-pk': u'ur',
+    u'vi-vn': u'vi',
+    u'wo-sn': u'wo',
+    u'yo-ng': u'yo',
+
+    # Locales that couldn't be directly mapped to the pontoon.
+    u'az-Latn-az': u'az',
+    u'bn-bd': u'bn-BD',
+    u'bn-in': u'bn-IN',
+    u'bs-Cyrl-ba': u'bs',  #CONFLICT?
+    u'bs-Latn-ba': u'bs',  #CONFLICT?
+    u'ca-ES-valencia': u'ca',
+    u'chr-Cher-US': u'',  # MISSING
+    u'en-gb': u'en-GB',
+    u'es-es': u'es-ES',
+    u'es-mx': u'es-MX',
+    u'fr-ca': u'fr',
+    u'gd-gb': u'gd',
+    u'guc-VE': u'',  # MISSING?
+    u'iu-Latn-ca': u'',  # MISSING?
+    u'kok-in': u'',  # MISSING?
+    u'ku-Arab-IQ': u'ku',
+    u'ky-kg': u'',  # MISSING?
+    u'lb-lu': u'',  # MISSING?
+    u'mi-nz': u'',  # MISSING?
+    u'mk-mk': u'mk',
+    u'mn-mn': u'mn',
+    u'ms-bn': u'ms', # CONFLICT?
+    u'ms-my': u'ms', # CONFLICT?
+    u'mt-mt': u'', # MISSING?
+    u'nb-no': u'nb-NO',
+    u'nn-no': u'nn-NO',
+    u'nso-za': u'nso', # CONFLICT?
+    u'or-in': u'or',  # CONFLICT?
+    u'pa-Arab-PK': u'', # MISSING?
+    u'prs-AF': u'',  # MISSING?
+    u'ps-af': u'',  # MISSING?
+    u'pt-br': u'pt-BR',
+    u'pt-pt': u'pt-PT',
+    u'quc-Latn-GT': u'',  # MISSING?
+    u'quz-pe': u'',  # MISSING?
+    u'rw-rw': u'',  # MISSING?
+    u'sd-Arab-PK': u'',  # MISSING?
+    u'sr-Cyrl-ba': u'sr',  # CONFLICT?
+    u'sr-Cyrl-rs': u'sr',  # CONFLICT?
+    u'sr-Latn-rs': u'sr',  # CONFLICT?
```

Comment:
We use Cyrillic script:
https://pontoon.mozilla.org/sr/

So use `sr-Cyrl-rs`.